src/main/java/me/squidxtv/frameui/core/Screen.java

```diff
@@ -47,6 +46,7 @@ public class Screen implements Clickable, Scrollable {
     private ClickEventHandler clickEventHandler = ClickEventHandler.empty();
     private ScrollEventHandler scrollEventHandler = ScrollEventHandler.empty();
 
+    private String name;
```

Comment:
It probably makes sense to make this final and let the user set it through the constructor/builder

Reply:
You are right, however adding next 5th or 6th parameter to Screen constructor is not the best idea. 

I thought about create class ScreenData that will holds meta data of the screen like 
- name 
- height 
- with 
- permissions
- etc..

By this we can reduce number of constructor parameters and make Screen class less crowded

What do you think of that?

Reply:
Yeah thought about that as well. Not sure if we want to name it `ScreenData` though, that kinda sounds like the visual data displayed on the screen.

Reply:
I can suggest to take inspiration from ItemStack and its data holder class ItemMeta. 
- ScreenMeta
- ScreenMetaData

or use web application convention 
- ScreenProperties 
- ScreenDataModel

However since english is not my primary language and you are the repo ower I will leave you picking the best name

Reply:
Okay, so English isn't my first language either. I am fine with the following three:
- `ScreenData`
- `ScreenMeta`
- `ScreenProperties`

Reply:
ScreenProperties won

Reply:
done